### ISSUE TITLE
Training parameters

### ISSUE THREAD

**Issue by kun-dragon** [TsingWei/LiteTrack#25]
How many batch sizes are there? And how many GPUs are used?
I use one A40 GPU, set the batch size to 128, and set the learning rate to 0.0004 (All changes in file "B4_cae_center_got10k_ep100.yaml").
But the performance of the tracker is: "AO: 61.8". It is significantly smaller than 65.2 (reported in your paper). The pretrained model is "cae_base_1600ep.pth".

**Comment by TsingWei** [comment]
Weird.  It's too old and I didn't save the training logs.  If you have time and still want to follow this project,  you can try different learning rates.
For lightweight and efficient scenarios, I recommend just using ViT-Tiny from https://github.com/wangsr126/mae-lite.  
In my experiments, it performed much better than the original 4-layer ViT-base, both in accuracy and latency.

**Comment by kun-dragon** [comment]
Thank you.
